Description: Don't install python-qpid, and set min version of six.
Author: Corey Bryant <corey.bryant@canonical.com>
Forwarded: not needed.
Index: heat/requirements.txt
===================================================================
--- heat.orig/requirements.txt	2014-11-27 15:38:15.102623924 +1100
+++ heat/requirements.txt	2014-11-27 15:38:27.934846752 +1100
@@ -8,7 +8,7 @@
 argparse
 lxml>=2.3
 netaddr>=0.7.6
-six>=1.6.0
+six>=1.5.2
 sqlalchemy-migrate>=0.8.2,!=0.8.4,!=0.9.2
 python-novaclient>=2.17.0
 PasteDeploy>=1.5.0
@@ -27,4 +27,3 @@
 paramiko>=1.9.0
 Babel>=1.3
 oslo.config>=1.2.0
-qpid-python
